libs/designer/src/lib/ui/panel/nodeDetailsPanel/nodeDetailsPanel.tsx

```diff
@@ -137,6 +147,8 @@ export const NodeDetailsPanel = (props: CommonPanelProps): JSX.Element => {
       selectTab={(tabId: string) => dispatch(selectPanelTab(tabId))}
       nodeId={selectedNode}
       readOnlyMode={readOnly}
+      canResubmit={runData?.canResubmit ?? false}
+      resubmitOperation={resubmitClick}
```

Comment:
If we move to remove "canSubmit"

```suggestion
      resubmitOperation={runData?.canResubmit ? resubmitClick: undefined}
```